src/message_transfer/domain/message.go

```diff
@@ -1,6 +1,23 @@
 package domain
 
+type id = uint16
+
 type Message struct {
-	Id     uint16
-	Fields map[string]string
+	id     id
+	values map[string]any
```

Comment:
Why do you used any? Messages only contain strings

Reply:
right now FW says they'll only send strings, but maybe in the future they might send numbers or booleans as well. By using any they can now send whatever they want in a message

Reply:
Let's stick to the specifications, we can change it in the future